chat-sdk-ui/src/main/java/co/chatsdk/ui/SendMultipleImagesAtOnceActivity.java

```diff
@@ -0,0 +1,181 @@
+package co.chatsdk.ui;
+
+import android.app.Activity;
+import android.content.ClipData;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.annotation.Nullable;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.github.chrisbanes.photoview.PhotoView;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.theartofdev.edmodo.cropper.CropImage;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import co.chatsdk.ui.chat.MediaSelector;
+import co.chatsdk.ui.main.BaseActivity;
+import co.chatsdk.ui.utils.Cropper;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+
+import static co.chatsdk.ui.chat.MediaSelector.CHOOSE_PHOTO;
+import static com.theartofdev.edmodo.cropper.CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE;
+
+public class SendMultipleImagesAtOnceActivity extends BaseActivity {
+
+    private String imageURI;
+    private ArrayList<Uri> uriArrayList;
+    private RecyclerView multipleImageRecycler;
+    private RecyclerView.LayoutManager layoutManager;
+
+    public PhotoView mainImageDisplay;
+    public Uri mainImageUri;
+    public ImageListAdapter imageListAdapter;
+    public File fileBeingCropped;
+    public Uri uriBeingCropped;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_send_multiple_images);
+
+        TextView topTextView = findViewById(R.id.topTextView);
+        TextView bottomTextView = findViewById(R.id.bottomTextView);
+        FloatingActionButton exit = findViewById(R.id.exit);
+        FloatingActionButton write = findViewById(R.id.write);
+        FloatingActionButton add = findViewById(R.id.add);
+        FloatingActionButton send = findViewById(R.id.send);
```

Comment:
These should be class variables. Also, they should be called like `exitButton` etc...